routing/missioncontrol.go

```diff
@@ -299,6 +258,39 @@ func (m *MissionControl) getEdgeProbabilityForNode(nodeHistory *nodeHistory,
 	return probability
 }
 
+// getPairProbability estimates the probability of successfully
+// traversing from fromNode to toNode based on historical payment outcomes.
+func (m *MissionControl) getPairProbability(fromNode,
```

Comment:
nice all-around refactor in this file 👍

Reply:
yes, a bit more isolation in a function helps to understand it easier.